modules/google-analytics/wp-google-analytics.php

```diff
@@ -0,0 +1,615 @@
+<?php
+/**
+ * Plugin Name: WP Google Analytics
+ * Plugin URI: http://bluedogwebservices.com/wordpress-plugin/wp-google-analytics/
+ * Description: Lets you use <a href="http://analytics.google.com">Google Analytics</a> to track your WordPress site statistics
+ * Version: 1.4.0
+ * Author: Aaron D. Campbell
+ * Author URI: http://ran.ge/
+ * License: GPLv2 or later
+ * Text Domain: wp-google-analytics
+ */
+
+define('WGA_VERSION', '1.4.0');
+
+/*  Copyright 2006  Aaron D. Campbell  (email : wp_plugins@xavisys.com)
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+*/
+
+/**
+ * wpGoogleAnalytics is the class that handles ALL of the plugin functionality.
+ * It helps us avoid name collisions
+ * http://codex.wordpress.org/Writing_a_Plugin#Avoiding_Function_Name_Collisions
+ */
+class wpGoogleAnalytics {
+
+	/**
+	 * @var wpGoogleAnalytics - Static property to hold our singleton instance
+	 */
+	static $instance = false;
+
+	static $page_slug = 'wp-google-analytics';
+
+	var $tokens = array();
+
+	/**
+	 * This is our constructor, which is private to force the use of get_instance()
+	 * @return void
+	 */
+	private function __construct() {
+		add_filter( 'init',                     array( $this, 'init' ) );
+		add_action( 'admin_init',               array( $this, 'admin_init' ) );
+		add_action( 'admin_menu',               array( $this, 'admin_menu' ) );
+		add_action( 'get_footer',               array( $this, 'insert_code' ) );
+		add_action( 'wp_enqueue_scripts',       array( $this, 'track_outgoing' ) );
+		add_filter( 'plugin_action_links',      array( $this, 'add_plugin_page_links' ), 10, 2 );
+	}
+
+ 	/**
+	 * Function to instantiate our class and make it a singleton
+	 */
+	public static function get_instance() {
+		if ( !self::$instance )
```

Comment:
Missing space and brackets here, as per coding standards:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style

It might help to use Code Sniffer with [the WordPress coding standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) to catch all those issues at once.